frontend/public/texts/organization_texts.js

```diff
@@ -279,6 +279,39 @@ export default function getOrganizationTexts({ organization, locale }) {
       en: "Skip for now",
       de: "Erstmal überspringen",
     },
+
+    add_social_media: {
+      en: "Add Social Media",
+      de: "Soziale Medien hinzufügen",
+    },
+    select_social_media: {
+      en: "Select a social media",
+      de: "Wähle ein Soziales Medium aus",
+    },
+    find_us_here: {
+      en: "Find us here:",
+      de: "Finde uns unter:",
+    },
+    does_not_comply_twitter: {
+      en: "Does not comply with a valid Twitter URL",
+      de: "Entspricht nicht einer gültigen Twitter-URL",
```

Comment:
```suggestion
      en: "This is not a valid Twitter URL",
      de: "Das ist keine gültige Twitter-URL",
```
Would do the same for all the things below as well